src/schema.py

```diff
@@ -144,9 +143,17 @@ def resolve_account_info(self, info, session_id=None):
       location = txn['locationName'].rsplit(' ', 1)[0]
       # removes the register numbers at the end of the string by taking the substring up until
       # the last space (right before the number)
+
+      if location == 'Olin Libe Cafe':
```

Comment:
why special-case this? you have it in `LOCATION_NAMES` too

Reply:
its because `Olin Libe Cafe` goes to `Amit Bhatia Libe Cafe` to match with the eatery name provided by cornell dining now, but it doesn't match the one we show on the screen.  We need a better way to organize the names in the future, but rn `Olin Libe Cafe` is just giving us some beef.